assets/js/base/hocs/with-component-id.js

```diff
@@ -8,9 +8,11 @@ import { Component } from 'react';
  */
 const withComponentId = ( OriginalComponent ) => {
 	let instances = 0;
-
 	class WrappedComponent extends Component {
-		instanceId = instances++;
+		constructor() {
+			super( ...arguments );
+			this.instanceId = instances++;
+		}
```

Comment:
I'm not sure to understand this change. Do we need to move this logic inside the constructor?

Reply:
Yes, it prevents the ID changing on re-render

Reply:
Wasn't that already fixed in #951?